test/azureCommand.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+// mock to test functionality in a command unrelated matter
+// this mean that not e.g. azureDeploy but the more abstract azureCommand can be used
+class AzureCommand {
```

Comment:
This is also a copy with a name replacement of the [Google plugin](https://github.com/serverless/serverless-google-cloudfunctions/blob/master/test/googleCommand.js)